docs/validation.md

```diff
@@ -46,6 +46,11 @@ Here are the available transforms:
   things currently in the fridge, and you want help the user to decide what to cook:
   `'ravioli' in 'fridge-contents'|answer|mapby('food-name')` will tell you whether
   there are ravioli.
+* `debug`: Does not modify the value, but writes the value to the log. This is
+  especially useful when you are exploring the data while building forms or
+  workflows. Depending on the environment, the log message may appear in
+  different places (Browser: console log in the debug tools, Server: System or
+  container logs, might depend on further configuration)
```

Comment:
Can we be more precise here, or at least link to someplace explaining "further configuration"?

Reply:
Clarified a bit, but I don't want to go into a full explanation as to where the log messages may go and what might influence it (depending on django config, whether you run in a container or not, within kubernetes or not, where is the log routed, is it collected or sent to a syslogd, etc)

Reply:
:+1: